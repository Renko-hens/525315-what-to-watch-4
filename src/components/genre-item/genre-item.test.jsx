import React from "react";
import renderer from "react-test-renderer";
import GenreItem from "./genre-item.jsx";

describe(`Renders Genre Item component`, () => {
  it(`Render component Genre Item`, () => {
    const tree = renderer.create(
        <GenreItem
          genre={`All genres`}
          activeGenre={`All genres`}
          onGenreLinkClick={() => {}}
        />
    ).toJSON;

    expect(tree).toMatchSnapshot();
  });
});
