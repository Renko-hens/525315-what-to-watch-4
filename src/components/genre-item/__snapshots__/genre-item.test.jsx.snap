// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Genre Item component Render component Genre Item 1`] = `[Function]`;
